<launch>
    <!-- flag if rviz should be started -->
    <arg name="use_rviz" default="true"/>

    <!-- lidar sensor -->
    <arg name="sensor_manufacturer" default="ouster"/>
    <arg name="sensor_model" default="os32"/>
    <arg name="sensor_position" default="left"/>
    <arg name="sensor_frame" default="sensor/lidar/$(arg sensor_model)_$(arg sensor_position)/os_sensor"/>
    <arg name="sensor_raw_data_topic" default="/bus/$(arg sensor_model)_$(arg sensor_position)/lidar_packets"/>

    <!-- manufacturer specific: ouster (use dummy values) -->
    <arg name="ouster_metadata_service" default="/bus/$(arg sensor_model)_$(arg sensor_position)/get_metadata"/>

    <!-- range image parameters -->
    <arg name="columns_per_full_rotation" default="1024"/> <!-- TODO: obtain this dynamically? -->

    <!-- clustering -->
    <include file="$(find streaming_clustering)/launch/streaming_clustering.launch" pass_all_args="true"/>
</launch>